app/models/form/sales/questions/buyer2_ethnic_group.rb

```diff
@@ -23,6 +23,6 @@ def initialize(id, hsh, page)
     "3" => { "value" => "Black, African, Caribbean or Black British" },
     "4" => { "value" => "Arab or other ethnic group" },
     "divider" => { "value" => true },
-    "17" => { "value" => "Buyer 2 prefers not to say" },
+    "17" => { "value" => "Buyer prefers not to say" },
```

Comment:
Would it make sense to remove `inferred_check_answers_value` here as well?

Reply:
I absolutely intended to - should be fixed now!